ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3058,3 +3058,50 @@ def test_tensorflow_Elu(
         features=x[0],
         name=name,
     )
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Conv2D",
+    x_f_d_df=_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NHWC", "NCHW"]),
+        padding=st.sampled_from(["SAME", "VALID"]),
```

Comment:
`padding` can also take the value  "EXPLICIT"

Reply:
sure will do that, now

Reply:
i should just add the "EXPLICIT" to the padding list right

Reply:
yes